src/audio/code_8019AF00.c

```diff
@@ -5052,11 +5367,21 @@ void Audio_SetCutsceneFlag(s8 flag) {
     sAudioCutsceneFlag = flag;
 }
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/code_8019AF00/func_801A3F6C.s")
+void Audio_PlaySfx_IfNotInCutsceneImpl(u16 sfxId, Vec3f* pos, u8 token, f32* freqScale, f32* volume, s8* reverbAdd) {
+    if (!sAudioCutsceneFlag) {
+        AudioSfx_PlaySfx(sfxId, pos, token, freqScale, volume, reverbAdd);
+    }
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/code_8019AF00/func_801A3FB4.s")
+void Audio_PlaySfx_IfNotInCutscene(u16 sfxId) {
+    Audio_PlaySfx_IfNotInCutsceneImpl(sfxId, &gSfxDefaultPos, 4, &gSfxDefaultFreqAndVolScale,
+                                      &gSfxDefaultFreqAndVolScale, &gSfxDefaultReverb);
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/code_8019AF00/func_801A3FFC.s")
+// Unused
+void Audio_MuteSfxAndAmbienceSeqExceptOcaAndSys(u8 arg0) {
```

Comment:
This is Ocarina and system right? I know its unused so it doesn't matter but two functions down writes both Ocarina and system out without abbreviation, which seems less necessary for an unused function.